getting-started/proc-configuring-dynamic-plugins-operator.adoc

```diff
@@ -0,0 +1,53 @@
+[id='proc-configuring-dynamic-plugins-operator_{context}']
+//= Configuring dynamic plugins in the {product} Operator
+
+.Procedure
+. Navigate to OpenShift and select the *ConfigMaps* tab.
+. Click *Create ConfigMap*.
++
+*Create ConfigMap* page appears.
+. Select the *YAML view* option in *Configure via* and edit the file, if needed.
+. Click *Create*.
+. Go to the *Topology* tab.
+. Click on the three dots of a {product-short} instance and select *Edit Backstage*.
++
+image::rhdh/rhdh-operator-edit.png[]
++
+This will load the YAML view of the {product-short} instance. 
+. Add the `spec.application.dynamicPluginsConfigMapName` field to the Custom Resource, as follows:
++
+[source]
+----
+spec:
+  application:
+ appConfig:
```

Comment:
Looks like there is an indentation issue here. Should be:
```yaml
spec:
  application:
    appConfig:
      mountPath: /opt/app-root/src
      configMaps:
        - name: app-config-rhdh
    dynamicPluginsConfigMapName: dynamic-plugins-rhdh
    extraEnvs:
      secrets:
        - name: secrets-rhdh
    extraFiles:
      mountPath: /opt-/app-root/src
    replicas: 1
    route:
      enabled: true
  database:
    enableLocalDb: true
```

Reply:
Indentation updated.